doc-content/enterprise-only/release-notes/rn-whats-new-con.adoc

```diff
@@ -3,149 +3,98 @@
-====
+In the *Task Inbox* and *Tasks* pages in {CENTRAL}, you can now perform bulk reassignment over multiple tasks in a single operation. To reassign tasks in bulk, select two or more tasks, click the *Bulk Actions* drop-down menu in the upper-right corner of the window, and select *Bulk Reassign*.
 
-=== Default filter selection
-You can use the *Saved Filter* option on the following pages to set a task filter as the default filter:
+In the *Tasks reassignment* window, enter the user identifier of the user to reassign the tasks, and click *Delegate*. For each task selected, a notification is displayed showing the reassignment result.
 
-* *Process Instances*
-* *Execution Errors*
-* *Jobs*
-* *Tasks*
-* *Task Inbox*
+=== Bulk actions supported for tasks in {CENTRAL}
 
-The default filter is then applied each time you open the page.
-endif::PAM[]
+In the  *Task Inbox* and *Tasks* pages in {CENTRAL}, you can now perform bulk actions over multiple tasks in a single operation. To update tasks in bulk, select two or more tasks, click the *Bulk Actions* drop-down menu in the upper-right corner of the window, and select one of the following bulk actions:
 
-=== Prometheus metrics monitoring in {PRODUCT}
+* *Bulk Claim*
+* *Bulk Release*
+* *Bulk Resume*
+* *Bulk Suspend*
 
-{KIE_SERVER} now supports Prometheus metrics monitoring for your business assets in {PRODUCT}. Prometheus is an open-source systems monitoring toolkit that you can use with {PRODUCT} to collect and store metrics related to the execution of business rules, processes, solvers, Decision Model and Notation (DMN) models, and other assets in {PRODUCT}. You can access the stored metrics through a REST API call to the {KIE_SERVER}, through the Prometheus expression browser, or using a data-graphing tool such as Grafana.
+If a specified bulk action is not permitted based on the task status, a notification is displayed and the operation is not executed on that task.
 
-You can configure Prometheus metrics monitoring for an on-premise {KIE_SERVER} instance, for {KIE_SERVER} on Spring Boot, or for a {KIE_SERVER} deployment on {OPENSHIFT}.
+=== Import and export Dashbuilder data in {CENTRAL}
 
-For information about enabling and using Prometheus metrics monitoring with {PRODUCT}, see {URL_MANAGING_KIE_SERVER}#prometheus-monitoring-con_execution-server[_{MANAGING_KIE_SERVER}_].
+You can now import and export Dashbuilder data as ZIP files. To export or import Dashbuilder data, select the *Admin* icon in the upper-right corner of the {CENTRAL} window and select *Dashbuilder Data Transfer*.
```

Comment:
Again, something minor. 1st you state import and export, then you reverse them in the next sentence. You r going to hurt me, aren't you?

Reply:
Good catch. You know I don't write this stuff, right? :-)